src/slider/slider.scss

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+
+  .iui-slider-min {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  .iui-slider-max {
+    margin-left: $iui-s;
+  }
+
+  &.iui-disabled {
+    @include iui-slider-disabled;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  z-index: 1; // above ticks
+  flex-grow: 1;
+  pointer-events: none;
+
+  .iui-tooltip {
+    display: none;
+  }
+
+  .iui-track {
+    pointer-events: none;  // let pointer event through to underlying range control
+    position: absolute;
+    width: 100%;
+    height: $track-height;
+    margin-top: 12px;
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+  }
+
+  .iui-slider-thumb {
+    outline: none;
+    position: absolute;
+    pointer-events: all;
+    height: $thumb-height;
+    width: $thumb-height;
+    border-radius: 100%;
+    top: $track-height + 1; // position center of thumb in center of track
```

Comment:
The regular slider's thumb is aligned correctly:
![Screen Shot 2021-07-08 at 11 03 08 AM](https://user-images.githubusercontent.com/849817/124945382-14d16b00-dfdc-11eb-99fd-897f5d0fa0e1.png)

But the ranges are off center, they are 2px further down than they should be.
![Screen Shot 2021-07-08 at 11 05 26 AM](https://user-images.githubusercontent.com/849817/124945715-67ab2280-dfdc-11eb-8e7b-7e8dd020ef8c.png)

I believe the `top` is the culprit, but changing it to fix the ranges then breaks the sliders. Why is the range getting more height than the slider?

Reply:
fixed